descriptions/api.github.com/api.github.com.yaml

```diff
@@ -17999,47 +17999,6 @@ paths:
         previews: []
         category: repos
         subcategory: statuses
-  "/repos/{owner}/{repo}/community/code_of_conduct":
```

Comment:
Was this route removed intentionally? @imwiss @xuorig

Reply:
Let me look into this and confirm.

Reply:
This has been reverted 👍  thanks for flagging it.